src/backend.f90

```diff
@@ -27,7 +27,6 @@ module m_base_backend
     real(dp) :: nu
     class(mesh_t), pointer :: mesh
     class(allocator_t), pointer :: allocator
-    class(dirps_t), pointer :: xdirps, ydirps, zdirps
```

Comment:
Is this related to adding the namelist?

Reply:
Not really, I just moved dirps instantiation inside solver and cleaned relevant things a bit. Also moved time integrator inside solver to simplify the input file namelist groups.